src/design-system/color/AccentColorContext.docs.tsx

```diff
@@ -43,7 +43,7 @@ const docs: DocsType = {
           source(
             <AccentColorProvider color="#ff0000">
               <Box background="accent" padding="19px (Deprecated)">
-                <Text size="17pt">
+                <Text color="label" size="17pt">
```

Comment:
Note that all usages within docs files like this one have been migrated to the new palette.